### PR TITLE
Resources

### PR DESCRIPTION
**Pull request by jsms90**
Please merge #120 first

### CODE REVIEW

File: coursebook/week-1/resources.md [hunk 1]
```diff
@@ -1,18 +1,32 @@
 ## Resources
 
-### Morning Challenges
+## Git
+[Udacity's git & GitHub course](https://www.udacity.com/course/how-to-use-git-and-github--ud775)
+Pro Git - the book [pdf](https://progit2.s3.amazonaws.com/en/2016-03-22-f3531/progit-en.1084.pdf) [website](https://git-scm.com/book/en/v2)
```
Comment (nelsonic) [inline coursebook/week-1/resources.md]: great resource. 👍 
could we point to the https://git-scm.com/book/en/v2 so people can _choose_ their format? 🤔

File: coursebook/week-1/resources.md [hunk 2]
```diff
@@ -1,18 +1,32 @@
 ## Resources
 
-### Morning Challenges
+## Git
+[Udacity's git & GitHub course](https://www.udacity.com/course/how-to-use-git-and-github--ud775)
```
Comment (bradreeder) [inline coursebook/week-1/resources.md]: Same issue as raised in #120 . These resources aren't appearing on new lines but in one block. Unsure if this is intentional.

File: coursebook/week-1/resources.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 ## Resources
 
+## Git
+[Udacity's git & GitHub course](https://www.udacity.com/course/how-to-use-git-and-github--ud775)
+Pro Git - the book [pdf](https://progit2.s3.amazonaws.com/en/2016-03-22-f3531/progit-en.1084.pdf) [website](https://git-scm.com/book/en/v2)
+[Comparing workflows tutorial](https://www.atlassian.com/git/tutorials/comparing-workflows)
+[Understanding the GitHub flow](https://guides.github.com/introduction/flow/)
```
Comment (bradreeder) [inline coursebook/week-1/resources.md]: Sorry to be annoying 😄 . Same problem as before, all the git resources are appearing on one line in a single block.

Comment (njsfield) [inline coursebook/week-1/resources.md]: No prob!

File: coursebook/week-1/resources.md (hunk 1 above)
Comment (jsms90) [inline coursebook/week-1/resources.md]: @nelsonic I don't understand. I did point to that website

Do you just mean that you would rather I removed the pdf option?

**Pull request merged**